src/stories/PageBanner.stories.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import PageBanner from '../components/PageBanner';
+import { Text, Button, ProgressBar } from 'components';
+
+storiesOf('PageBanner', module).add('with ProgressBar', () => (
+  <div>
+    <Text type="H3">PageBanner With ProgressBar, Edit and Close Buttons</Text>
+    <br />
+    <br />
+    <PageBanner>
+      <span style={{ width: '50px' }} />
+      <ProgressBar heading="Profile Strength" percentage={75} />
+      <span style={{ width: '20px' }} />
+      <span className="width-stretch">
+        <Button onClick={action('Edit clicked')} type="link">
+          Edit Profile
+        </Button>
+        <span style={{ float: 'right' }}>
+          <Button onClick={action('Close clicked')} type="link">
+            x
+          </Button>
+        </span>
+      </span>
+    </PageBanner>
+    <br />
```

Comment:
Spamming br tags is actually a really bad practice as it breaks very easily, isn't as dynamic, and bloats the codebase. It is so much better adjust the spacing using styling like margins and padding. Could you remove these and achieve the intended effect with styling?